timesketch/lib/charts/interface.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Interface for charts."""
+
+from __future__ import unicode_literals
+
+import altair as alt
+
+
+class BaseChart(object):
+
+    NAME = 'name'
```

Comment:
maybe add a comment here what this is all about, what does this mean

Reply:
Done.